source/contributing.md

```diff
@@ -0,0 +1,174 @@
+
+### Edit documentation online with GitHub
+
+Select the **Edit in GitHub** link on the sidebar. The source `.rst` file of the page opens in a GitHub editor in your browser. If you haven't worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).
+
+Edit the page. You can review your changes on the **Preview** tab.
+
+  **NOTE**: GitHub's native markup language is [Markdown](https://daringfireball.net/projects/markdown/). GitHub's Markdown renders titles, headings, and lists correctly but does not render RestructuredText-specific elements such as directives or the variable class.
+
+Commit your changes. 
+
+In the **Commit changes** section, add the following:
+
+  - Brief summary (72 characters or less) of what was changed
+  - Details about why and what changed, if needed.
+  
+Then select the **Create a new branch for this commit and start a pull request** option. Accept the name for the branch and then Select **Commit changes**.
+
+GitHub creates a branch and a commit for your changes. It loads a new page on which you can submit a pull request to Percona. The page shows the base branch - the one you offer your changes for, your commit message and a diff - a visual representation of your changes against the original page. At this time you can make a last-minute review. When you are ready, click the **Create pull request** button.
+
+A core developer reviews the pull request for accuracy and merges your pull request into the repository, if everything is correct. The documentation  gets published on the site at the next release.
+
+**NOTE**: The pull request may be redone to meet internal requirements, for example, aligning the Percona branch to the upstream branch.
+
+### Edit documentation locally
+
+This option is for users who prefer to work from their computer and/or have the full control over the documentation process.
+
+The steps are the following:
+
+1. Fork this repository
+
+2. Clone the forked repository to your machine:
+
+  ```bash
+   git clone git@github.com:<your_name>/proxysql-admin-tool-doc.git
+   cd <directory name>/proxysql-admin-tool-doc
+  ```
+
+3. Add the remote origin repository:
+
+  ```sh
+   git remote add origin 
+   https://github.com/percona/proxysql-admin-tool-doc.git
+  ```
+
+4. Checkout the appropriate branch and pull the latest changes from origin
+
+  ```sh
+   git checkout main && git pull origin main
+  ```
+
+5. Create a separate branch for your changes
+
+  ```sh
+   git checkout -b <my_changes>
+  ```
+
+6. Do your work. Add code examples, if necessary. We recommend that you build an HTML file, either [locally](#Install Sphinx and build locally) or [using Docker](# Use Docker) to view your changes. You can run the command to `make clean html` to see the latest changes. If you have questions on formatting, use a [cheat sheet](https://sphinx-tutorial.readthedocs.io/cheatsheet/?highlight=-b#rst-cheat-sheet).
```

Comment:
Links won't work in this folrmat. They require no spaces between # and the link text, plust use - as a separator

Reply:
Fixed.